### PR TITLE
Use scale in transforamtion for Notification item on hover

### PR DESCRIPTION
**Pull request by Link2Twenty**
Transforms are handed to the GPU so can be processed faster.

### CODE REVIEW

File: var/httpd/htdocs/skins/Agent/default/css/Core.Header.css [hunk 1]
```diff
@@ -434,14 +434,17 @@ body.Visible-ScreenXL #ToolBar li.UserAvatar div a:hover {
...
+    width: 100%;
     height: 3px;
     position: absolute;
     bottom: 0px;
     left: 0px;
     background: #f92;
     content: "";
-    transition: width ease 0.3s;
+    transform: scale(0, 1);
+    transform-origin: 0;
```
Comment (Link2Twenty) [inline var/httpd/htdocs/skins/Agent/default/css/Core.Header.css]: If you remove this line the animation would spread from the centre, which, might look better.

**Comment by s7design** [comment]
Hi,

Thanks for PR.

If you would like to make new PR, what is welcomed always, please subscribe OTRS contributor agreement on the site http://doc.otrs.com/

Regards
Zoran

**Comment by s7design** [comment]
Hi, 

It is applied and it will be available from OTRS 6.0.15. 

There is commit:
https://github.com/OTRS/otrs/commit/1c30d5c96ab2cef26bff1eb4dcd096a739be2521

Regards
Zoran

**Comment by Link2Twenty** [comment]
Thank you, I've sent the CLA now too 😀

**Pull request closed without merging**